Source/WebCore/WebCoreMacros.cmake

```diff
@@ -57,7 +57,7 @@ option(SHOW_BINDINGS_GENERATION_PROGRESS "Show progress of generating bindings"
 #   SUPPLEMENTAL_DEPFILE is a value of --supplementalDependencyFile. (optional)
 #   PP_EXTRA_OUTPUT is extra outputs of preprocess-idls.pl. (optional)
 #   PP_EXTRA_ARGS is extra arguments for preprocess-idls.pl. (optional)
-function(GENERATE_BINDINGS target)
+macro(GENERATE_BINDINGS target)
```

Comment:
Isn't it possible to keep this as a function to avoid polluting the global namespace with the local variables used in the function? If you need to set a value globally, you can use `set(variablename value PARENT_SCOPE)`, maybe that would be enough.

Reply:
Done.